packages/core/src/features/catalog/opening-entity-details.test.tsx

```diff
@@ -129,6 +133,7 @@ describe("opening catalog entity details panel", () => {
     describe("when opening the menu 'some-kubernetes-cluster'", () => {
       beforeEach(() => {
         rendered.getByTestId("icon-for-menu-actions-for-catalog-for-some-entity-id").click();
+        advanceFakeTime(1000);
```

Comment:
Is the 1000ms something special here? The name of describe doesn't really provide any information and I would have to dig deeper in the code. What happens after 999ms?

Reply:
This is to cover animations